app/js/app.js

```diff
@@ -72,3 +72,9 @@ appendChildController.mainController = mainController;
 moveController.mainController = mainController;
 
 window.mainController = mainController;
+
+// We insert the main view into the document on startup.  This adds a
+// small disabled stylesheet and a minimal offscreen element to the
+// DOM. The mainView keeps the rest of the customizer out of the DOM
+// until its open() method is called.
+document.body.appendChild(mainView.el);
```

Comment:
Okay, I'll move that back. I did it this way because in this app, none of the views insert themselves. The main view is in charge of inserting all of the child views, so I figured that I should go up a level from main view to insert the that.  I'll just put it in main controller, then.